src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -49,6 +49,7 @@
     private static final String CEUTRIO_20_21_EXPECTED_VCF = "CEUTrio.20.21.gatk3.7_30_ga4f720357.expected.vcf";
     private static final File NA12878_HG37 = new File(toolsTestDir + "haplotypecaller/expected.testGVCFMode.gatk4.g.vcf");
     private static final List<String> ATTRIBUTES_WITH_JITTER = Arrays.asList(
+            "PS",
```

Comment:
We use the jitter set for things that have numerical precision issues.  Please move this to ATTRIBUTES_TO_IGNORE.